### PR TITLE
Fixes to documentation regarding plugins

### LINKED ISSUE #5938: Typo in docs relating to example code for worker plugins
**Issue**:
From plugins.rst, there's an example illustrating the use of workerplugins
```python
class ErrorLogger(WorkerPlugin):
    def __init__(self, logger):
        self.logger = logger
...
```
```python
plugin = ErrorLogger()
client.register_worker_plugin(plugin)
```
However the code incorrectly instantiates ErrorLogger since it's missing the actual logger

**Solution**
```python
import logger
plugin = ErrorLogger(logger)
```
This requires a minimal change and thus I intend on submitting an MR shortly.

### PR DESCRIPTION
**Pull request by crendoncoiled**
- [x] Closes #5938
- [x] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: distributed/diagnostics/plugin.py [hunk 1]
```diff
@@ -127,7 +127,8 @@ class WorkerPlugin:
     ...                 exc_info=exc_info
     ...             )
 
-    >>> plugin = ErrorLogger()
+    >>> import logger
```
Comment (ncclementi) [inline distributed/diagnostics/plugin.py:130]: Hi @crendoncoiled thanks for the PR, I'm just trying to run the example but `import logger` doesn't work. I believe this is `import loggin` so maybe we can re-write this as. 

```suggestion
    >>> import logging as logger
```

Comment (crendoncoiled) [reply distributed/diagnostics/plugin.py]: I see the typo! Fixed

**Comment by ncclementi** [comment]
@crendoncoiled Thank you for your contribution. I believe all the failures on CI are unrelated but we need one of the maintainers to take a final look and merge this in. 

cc: @fjetter

**Review by jcrist — APPROVED** [review]
Thanks!

**Pull request merged**